FRINXio/frinx-frontend
# add device details to GraphNode

You need to run this https://github.com/FRINXio/frinx-inventory-server/pull/237.

Add `deviceType` and `softwareVersion` to device detail panel.

HOW TO TEST:
- open device topology
- click on a few nodes to display the info panel
- you should see those two values there (they are not present in every node)
